source/tutorials/kubernetes.rst

```diff
@@ -3,170 +3,340 @@
 Kubernetes\*
 ############
 
-This tutorial describes how to install, configure, and start the
-`Kubernetes container orchestration system`_ on |CL-ATTR| using CRI+O and kata-runtime.
+This tutorial describes how to install, configure, and start the `Kubernetes
+container orchestration system`_ on |CL-ATTR| using a variety of `container
+runtimes
+<https://kubernetes.io/docs/setup/production-environment/container-runtimes/>`_.
+
+Alternatively, you can get a Kubernetes cluster up and running quickly with
+our `cloud-native-setup repository`_ which includes scripts to automates
+cluster initialization.
 
 .. contents::
    :local:
    :depth: 1
 
-Description
+Background
 ***********
 
-Kubernetes is an open source system for automating deployment, scaling, and
-management of containerized applications. It groups containers that make up
-an application into logical units for easy management and discovery. Get up
-and running quickly with our `Cloud native setup automation`_.
+|CL| has builtin integrations to make setting up Kubernetes using a variety of
+container runtimes.
+
+For more background information see:
 
-Kata Containers\* kata-runtime adheres to
-:abbr:`OCI (Open Container Initiative*)` guidelines and works seamlessly with
-Kubernetes. `Kata Containers`_ provides strong isolation for untrusted
-workloads or  multi-tenant scenarios. Kata Containers can be
-allocated on a per-pod basis, so you can mix and match both on the same host
-to suit your needs.
+* `What is Kubernetes?`_
+* `What is CRI+O?`_
```

Comment:
This feels incomplete without containerd and Docker. If you wanted to follow how Kubernetes describes these at https://kubernetes.io/docs/setup/production-environment/container-runtimes/ you would have   Docker/CRI-O/Containerd/Frakti which are CRI runtime replacements.  Kata is an OCI runtime and Docker uses runc which is also an OCI runtime.

Reply:
I agree, it does seem out of place. I originally pulled out sections based on them being too much background information embedded in sections.  CRI+O was one of these.

Looking at it now, I actually think it would be better to simplify this to just 2: What is Kubernetes? and What is Kata?. The rest of the container runtime descriptions are small enough to embed, I think.

Reply:
Makes sense.  I think either you need a comprehensive list or a very focused list.  Putting some CRI runtimes and not others is confusing.  I have a similar comment at the end of the doc where these link to.